README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
-in the Service Manual][service-manual-assistive-technologies].
+- is accessible for users who only use a keyboard
+- lets users adjust or override colours in Windows, Firefox and Chrome
+- is usable in Internet Explorer 8, 9 and 10
 
-Currently, GOV.UK Frontend officially supports the following assistive technologies:
+### Accessibility issues and warnings
 
-| Software                 | Version        | Type               | Browser                   |
-|--------------------------|----------------|--------------------|---------------------------|
-| JAWS                     | 15 or later    | screen reader      | Internet Explorer 11      |
-| ZoomText                 | 10.11 or later | screen magnifier   | Internet Explorer 11      |
-| Dragon NaturallySpeaking | 11 or later    | speech recognition | Internet Explorer 11      |
-| NVDA                     | 2016 or later  | screen reader      | Firefox (latest versions) |
-| VoiceOver                | 7.0 or later   | screen reader      | Safari on iOS10 and OS X  |
+We're working on fixing the following issues from our [issues list in GitHub](https://github.com/alphagov/govuk-frontend/issues/).
 
-In addition, we test that all content is accessible with keyboard only.
+- `legend` elements are not visible to some assistive technologies.
+- Radios with [conditionally revealing content](https://design-system.service.gov.uk/components/radios/#conditionally-revealing-content) fail when they're tested with [Axe](https://github.com/dequelabs/axe-core).
+- The date input component uses `type="number"` and `role="group"`, which causes several accessibility issues.
+- The navigation in the page header uses `aria-label` wrongly, which may cause issues with assistive technologies.
 
-We aim to support [users who adjust or override the colours of websites they visit][how-users-change-colours-on-websites]. We test this by [changing colours in Firefox][changing-colours-in-firefox], by [enabling 'High Contrast' mode in Windows][enabling-high-contrast-mode-in-windows] and by using the [High Contrast plugin for Chrome][high-contrast-plugin-for-chrome].
+Accessibility checkers may also warn you about the following issues. Read about [why these issues should not affect users](https://github.com/alphagov/govuk-frontend/issues/1280).
 
-[service-manual-assistive-technologies]: https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test
-
-[changing-colours-in-firefox]:
-https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use
-
-[enabling-high-contrast-mode-in-windows]:
-https://support.microsoft.com/en-gb/help/13862/windows-use-high-contrast-mode
-
-[high-contrast-plugin-for-chrome]: https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en-US
-
-[how-users-change-colours-on-websites]:
-https://accessibility.blog.gov.uk/2017/03/27/how-users-change-colours-on-websites/
+- `aria-disabled` is invalid in disabled elements.
```

Comment:
This list doesn't currently tell you which component the warnings relate to (it's not clear for many of them), or the reason for why it happens.

Like Nick suggests we should probably link to these in issue tracker so that the context/reasoning is available.

Reply:
I'm not sure linking alone will work here for a couple of reasons - please see my comment here: https://github.com/alphagov/govuk-frontend/pull/1616#discussion_r337034374

Reply:
Thanks @m-green I've responded in #1616 (comment)